README.rst

```diff
@@ -1,19 +1,16 @@
 ==========
-``ci_hsc``
+``gen2_ci_hsc``
 ==========
 
-``ci_hsc`` provides test data and script which exercises the LSST stack
+``gen2_ci_hsc`` provides test data and script which exercises the LSST stack
 through single frame and coadd processing based on engineering test data from
 Hyper Suprime-Cam.
 
 Obtaining test data
 ===================
 
-The data used by ``ci_hsc`` is stored using `Git LFS`_; refer to the `relevant
-LSST documentation`_ for details on how to check out this repository.
-
-.. _Git LFS: https://git-lfs.github.com
-.. _relevant LSST documentation: http://developer.lsst.io/en/latest/tools/git_lfs.html
+The data used by ``gen2_ci_hsc`` linked against data installed by ``ci_hsc``, please
```

Comment:
```suggestion
The data used by ``ci_hsc_gen2`` linked against data installed by ``ci_hsc``, please
```

Reply:
I'm a little confused why data is needed from both `ci_hsc` and `testdata_ci_hsc`, or if this should just be updated to say `testdata_ci_hsc`. I would naively assume the new existence of the latter means the former is obsolete. If this is *not* right, please explain why.

Reply:
it should say testdata_ci_hsc, the name change to testdata occurred late and I missed this change.